spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/BlockingQueueConsumer.java

```diff
@@ -483,8 +518,11 @@ private void attemptPassiveDeclarations() {
 				if (logger.isWarnEnabled()) {
 					logger.warn("Failed to declare queue:" + queueName);
 				}
+				if (!this.connection.isOpen()) {
```

Comment:
Here is the simple polishing https://github.com/artembilan/spring-amqp/commit/a2cd0782e6480240d5e78b7abfed99789654da51
However I want to know why `this.channel.isOpen()` isn't enough here?
Looks like closing connection causes close for all its channels. See `AMQConnection.finishShutdown`